contracts/SafeL2.sol

```diff
@@ -69,15 +74,15 @@ contract SafeL2 is Safe {
     }
```

Comment:
Not in this PR, but can we follow the same patter for `execTransaction`? I'm not a huge fan of the inconsistency with how we are emitting new events in `SafeL2`.